sprite.go

```diff
@@ -325,6 +414,25 @@ func cloneSprite(out reflect.Value, outPtr Spriter, in reflect.Value, v specsp)
 	return dest
 }
 
+func Gopt_SpriteImpl_Clone__0(sprite Spriter) {
+	Gopt_Sprite_Clone__1(sprite, nil)
+}
+
+func Gopt_SpriteImpl_Clone__1(sprite Spriter, data interface{}) {
+	src := spriteOf(sprite)
+	if debugInstr {
+		log.Println("Clone", src.name)
+	}
+	in := reflect.ValueOf(sprite).Elem()
+	v := reflect.New(in.Type())
+	out, outPtr := v.Elem(), v.Interface().(Spriter)
+	dest := cloneSprite(out, outPtr, in, nil)
+	src.g.addClonedShape(src, dest)
+	if dest.hasOnCloned {
+		dest.doWhenCloned(dest, data)
+	}
+}
+
 func Gopt_Sprite_Clone__0(sprite Spriter) {
```

Comment:
I think we should remove `Gopt_Sprite_Clone__*`?

Reply:
done